server/src/repositories/album-permission.repository.ts

```diff
@@ -0,0 +1,32 @@
+import { Injectable } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Equals } from 'class-validator';
+import { AlbumPermissionEntity } from 'src/entities/album-permission.entity';
+import { IAlbumPermissionRepository } from 'src/interfaces/album-permission.interface';
+import { Instrumentation } from 'src/utils/instrumentation';
+import { Equal, Repository } from 'typeorm';
+
+@Instrumentation()
+@Injectable()
+export class AlbumPermissionRepository implements IAlbumPermissionRepository {
```

Comment:
I might just put this in the album repository?
- addSharedUser
- updateSharedUser 
- removeSharedUser

Reply:
I would prefer to keep it, I think it's more readable this way.